public/app/models/finding_aid_pdf.rb

```diff
@@ -103,11 +127,19 @@ def source_file
 
 
         ao_list.each do |record, entry, is_parent|
-          writer.write(renderer.render_to_string partial: 'archival_object', layout: false, :locals => {:record => record, :level => entry.depth, :is_parent => is_parent})
+          begin
+            writer.write(renderer.render_to_string partial: 'archival_object', layout: false, :locals => {:record => record, :level => entry.depth, :is_parent => is_parent})
+          rescue => e
+            raise PDFRenderErrorArchivalObject, "#{e.class};#{e.message}"
```

Comment:
Also here, since we know the record details, we could include the title and / or id of the record in the error message that the user sees, to help them identify where the error is.

Reply:
Added title to the error report to assist user.